app/models/harvest.rb

```diff
@@ -9,7 +9,6 @@ class Harvest < ActiveRecord::Base
   belongs_to :plant_part
   belongs_to :planting
 
-  default_scope { order('created_at DESC') }
```

Comment:
Related: https://github.com/rails/rails/issues/5701
Unscoped doesn't take away the default scope's order by, but reorder did the trick.